piet-web/src/text.rs

```diff
@@ -172,6 +172,11 @@ impl TextLayout for WebTextLayout {
         self.size
     }
 
+    fn image_bounds(&self) -> Rect {
+        //FIXME: figure out actual image bounds on web?
```

Comment:
Yes, there are methods for this, see https://developer.mozilla.org/en-US/docs/Web/API/TextMetrics (which it might be valid to link). I think the image bounds methods are implemented on all browsers which can reasonably run Rust, but for good measure it's probably best to sense at runtime.

Reply:
going to leave this in and I'll open an issue.